spiderable_server.js

```diff
@@ -16,6 +16,13 @@ var crypto = Npm.require('crypto');
 Spiderable.userAgentRegExps = [
     /^facebookexternalhit/i, /^linkedinbot/i, /^twitterbot/i];
```

Comment:
If bingbot is a common bot, then I'd say it should also be added to this list, along with google, I suppose.

Also, move the close bracket to the next line for readability. Consider making each bot on one line.